source/features/sticky-sidebar.tsx

```diff
@@ -13,7 +13,7 @@ const minimumViewportWidthForSidebar = 768; // Less than this, the layout is sin
 const sidebarSelector = [
 	'.Layout-sidebar .BorderGrid', // `isRepoRoot`
 	'.Layout-sidebar #partial-discussion-sidebar', // Old `isConversation`
-	'div[data-testid="issue-viewer-metadata-pane"]', // `isConversation`
+	'div[data-testid="issue-viewer-metadata-pane"]:not(:has([data-testid="sticky-sidebar"]))', // `isConversation`
```

Comment:
`:has` is always a last resort. Here you most likely want to disable the feature altogether when native sticky sidebars are detected.

Reply:
The way I've been doing it is to add a secondary `observe('[data-testid="sticky-sidebar"]', () => features.unload(this one))`. Alternatively you could add `if (containsElement(foundSidebar)) return` in `trackSidebar` (preferable).